packages/firebase/compat/index.d.ts

```diff
@@ -6018,6 +6018,25 @@ declare namespace firebase.database {
      * ```
      */
     app: firebase.app.App;
+    /**
+     * INTERNAL methods for internal-use only (tests, etc.).
+     *
+     * @const
```

Comment:
What does this tag do?

Reply:
Defines a constant, but it isn't supported by TypeScript